app/lib/cost_calculator.rb

```diff
@@ -22,11 +22,7 @@ def letter_and_call_cost(object, scope)
     end
 
     def disbursement_cost(object)
-      if object.disbursement_type.value == 'other'
-        object.total_cost_without_vat
-      else
-        object.miles * object.pricing
-      end
+      object.provider_requested_total_cost_without_vat + object.vat_amount
```

Comment:
so I think this will return the incorrect value for the provider requested total cost when the disbursement is disallowed as it is using the `current` value of the `vat_amount` when it should be the `provider_requested_vat_amount`

Reply:
Provider don't request the vat_amount for Disbursement items. It gets applied based on the items. In JSON we get 0 as vat_amount if item doesn't qualify or the calculated vat_amount if it does. Happy to have a chat if this doesn't make any sense

Reply:
system spec added

Reply:
so the issue is we are using the provider requests total cost and the current vat_amount

This is inconsistent as when the caseworker has disallowed the disbursement the vat amount for the provider requested amount is adjusted to 0.